test/jdk/javax/swing/JSplitPane/4820080/JSplitPaneDragColorTest.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2003, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.Robot;
+import java.awt.Toolkit;
+import java.awt.event.InputEvent;
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
+import javax.imageio.ImageIO;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JSplitPane;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UIManager.LookAndFeelInfo;
+import javax.swing.UnsupportedLookAndFeelException;
+
+import static javax.swing.UIManager.getInstalledLookAndFeels;
+
+/*
+ * @test
+ * @key headful
+ * @bug 4820080
+ * @summary This test confirms that the Drag color of JSplitPane divider should be the user specified one(Red here).
+ * @run main JSplitPaneDragColorTest
+ */
+public class JSplitPaneDragColorTest {
+
+    // Tolerance is set inorder to negate small differences in pixel color values, especially in Mac machines.
+    private final static int COLOR_TOLERANCE = 9;
+    private static final Color EXPECTED_DRAG_COLOR = Color.RED;
+    private static JFrame frame;
+    private static JSplitPane pane;
+    private static Robot robot;
+
+    public static void main(String[] args) throws Exception {
+
+        robot = new Robot();
+        robot.setAutoWaitForIdle(true);
+        robot.setAutoDelay(200);
+        // Skipping NimbusLookAndFeel & GTKLookAndFeel as both are not supported for this feature - JDK-8075914
```

Comment:
Please split the long lines using 80 chars per line.
Note that JDK-8075914 is closed so it is unclear why this is unsupported by Nimbus/GTK l&f.

Reply:
I think the closed bug is JDK-8075608, but JDK-8075914 is still open. But anyway JDK-8075608 was closed as 'Won't Fix', so I think the problem remains.

Reply:
Fixed the long lines issue.